aptos-move/framework/table-natives/src/lib.rs

```diff
@@ -129,11 +141,17 @@ impl<'a> NativeTableContext<'a> {
                 match op {
                     Op::New(val) => {
                         let bytes = serialize(&value_layout_info.layout, &val)?;
-                        entries.insert(key, Op::New(bytes.into()));
+                        let layout = value_layout_info
+                            .has_aggregator_lifting
+                            .then(|| value_layout_info.layout.clone());
+                        entries.insert(key, Op::New((bytes.into(), layout)));
                     },
                     Op::Modify(val) => {
                         let bytes = serialize(&value_layout_info.layout, &val)?;
-                        entries.insert(key, Op::Modify(bytes.into()));
+                        let layout = value_layout_info
+                            .has_aggregator_lifting
+                            .then(|| value_layout_info.layout.clone());
```

Comment:
Ditto

Reply:
Tried it now. It then gives the borrow of moved value error.